File: app/src/main/java/vergecurrency/vergewallet/service/repository/db/ContactRepositoryDAO.java
package vergecurrency.vergewallet.service.repository.db;

import org.greenrobot.greendao.annotation.Entity;
import org.greenrobot.greendao.annotation.Id;
import org.greenrobot.greendao.annotation.Property;

@Entity(nameInDb = "contact")
public class ContactRepositoryDAO {

    @Id(autoincrement = true)
    private long id;
    @Property(nameInDb = "name")
    private String name;
    @Property(nameInDb = "address")
    private String address;



}

File: app/src/main/kotlin/vergecurrency/vergewallet/service/repository/db/ContactRepositoryDAO.kt
package vergecurrency.vergewallet.service.repository.db

import org.greenrobot.greendao.annotation.Entity
import org.greenrobot.greendao.annotation.Id
import org.greenrobot.greendao.annotation.Property

@Entity(nameInDb = "contact")
class ContactRepositoryDAO {

    @Id(autoincrement = true)
    private val id: Long = 0
    @Property(nameInDb = "name")
    private val name: String? = null
    @Property(nameInDb = "address")
    private val address: String? = null


}
